src/content/cloud/deploy-from-git.mdx

```diff
@@ -17,13 +17,13 @@ Please make sure to configure your Git repositories with the [appropriate manife
 
 ## Deploy
 
-Log in to [Okteto Cloud](https://cloud.okteto.com), and click on the **Launch Dev Environment** button on the top left.
+Log in to your Okteto instance (e.g. https://cloud.okteto.com), and click on the **Launch Dev Environment** button on the top left.
```

Comment:
```suggestion
Log in to your Okteto instance (e.g. https://example-dev.okteto.com), and click on the **Launch Dev Environment** button on the top left.
```

We can use another example, but I thought moving away from `cloud` even in these mentions might be helpful to reduce points of confusion